src/test/features/claimant-response/draft/draftClaimantResponse.ts

```diff
@@ -44,7 +51,13 @@ describe('DraftClaimantResponse', () => {
           option: PaidAmountOption.YES,
           amount: 999,
           claimedAmount: 1000
-        }
+        },
```

Comment:
I would suggest some negative testing where not all values are provided - will the deserialize still work?

Reply:
added a deserialise test with partial data ...there are 2 existing negative tests for undefined and null